mobile_verifier/src/cli/reward_from_db.rs

```diff
@@ -38,8 +39,17 @@ impl Cmd {
         let heartbeats = HeartbeatReward::validated(&pool, &epoch);
         let speedtest_averages =
             SpeedtestAverages::aggregate_epoch_averages(epoch.end, &pool).await?;
-        let reward_shares =
-            CoveragePoints::aggregate_points(&pool, heartbeats, &speedtest_averages, end).await?;
+        let boosted_hexes = BoostedHexes {
+            hexes: HashMap::new(),
+        };
```

Comment:
Give BoostedHexes as a default impl and avoid making its members public

Reply:
was already there, hadnt thought to use it..fixed